test/config-next/cert-checker.json

```diff
@@ -9,7 +9,7 @@
     "unexpiredOnly": true,
     "badResultsOnly": true,
     "checkPeriod": "72h",
-    "acceptableValidityPeriods": [7776000],
+    "acceptableValidityDurations": ["7775999s", "7776000s"],
```

Comment:
We should still only be specifying 7776000s here; the need for 7775999 has passed.